apps/parsley/src/utils/errorReporting/errorReporting.test.ts

```diff
@@ -9,13 +10,17 @@ import {
 const { cleanup, mockEnv } = mockEnvironmentVariables();
 
 describe("error reporting", () => {
+  beforeAll(() => {
+    overwriteFakeTimers();
+  });
```

Comment:
Is there a way to call `overwriteFakeTimers` at the global level so we never have to call this function within individual tests?